src/js/component.js

```diff
@@ -524,6 +525,52 @@ class Component {
     return currentChild;
   }
 
+  /**
+   * Adds an SVG icon element to another element or component.
+   *
+   * @param {string} iconName
+   *        The name of icon. A list of all the icon names can be found...
+   *
+   * @param {Component|Element} [el=this.el()]
+   *        Component or element to set the title on. Defaults
+   *
+   * @return {Element}
+   *        The newly created icon element.
+   */
+  setIcon(iconName, el = this.el()) {
+    if (!this.player_.options_.useSVGIcons) {
+      return;
+    }
```

Comment:
Putting this here means it's easy to remove in Video.js 9 - good thinking. Maybe a TODO comment here to remove this in Video.js 9?